test/runtests.jl

```diff
@@ -124,11 +124,11 @@ MT4_2{Float64}(4,5)
 @test_throws ErrorException MT4_2{Float64}()
 
 # user defined inner positional constructor
-@with_kw immutable MT6{R,I<:Integer} <: AMT{R}
+@with_kw struct MT6{R,I<:Integer} <: AMT{R}
     r::R=5
     a::I
-    @compat (::Type{MT6{R,I}}){R,I}(r) = new{R,I}(r,r)
-    @compat (::Type{MT6{R,I}}){R,I}(r,a) = (@assert a>r; new{R,I}(r,a))
+    (::Type{MT6{R,I}})(r) where {R,I} = new{R,I}(r,r)
+    (::Type{MT6{R,I}})(r,a) where {R,I} = (@assert a>r; new{R,I}(r,a))
```

Comment:
bad bot

Reply:
Note how this line did not get translated but the one here https://github.com/mauro3/Parameters.jl/pull/40/files#diff-fce720c43af3c52c862fd7451c7374b8R169 did.  The only difference is that the translated one is in an expression.